CMakeLists.txt

```diff
@@ -193,6 +193,13 @@ set (Boost_MULTITHREADED TRUE)
 
 if (USE_STATIC_BOOST)
     set (Boost_USE_STATIC_LIBS TRUE)
+    if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
Now it occurs to me: can this be done in `win-vs.txt` instead of here? That's where Windows-specific setup should reside, ideally...